### PR TITLE
Added the installation guide for Ubuntu

### PR DESCRIPTION
**Pull request by Kumar-Kishan**
Added installation guide

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,78 @@ IRC channel: https://freenode.net #googlebenchmark
 
 [Additional Tooling Documentation](docs/tools.md)
 
+
+## Installation Guide
+First make sure you have git and cmake installed (If not please install it)
+
+```
```
Comment (dominichamon) [inline README.md]: make sure it's clear this bit is for ubuntu/debian.

File: README.md [hunk 2]
```diff
@@ -13,6 +13,78 @@ IRC channel: https://freenode.net #googlebenchmark
...
+Now, let's clone the repository and build it
+
+```
+git clone https://github.com/google/benchmark.git
+cd benchmark
+mkdir build
+cd build
+cmake .. -DCMAKE_BUILD_TYPE=RELEASE
+make
+cmake ..
```
Comment (dominichamon) [inline README.md]: why do you need the second cmake?

File: README.md [hunk 3]
```diff
@@ -13,6 +13,78 @@ IRC channel: https://freenode.net #googlebenchmark
...
+```
+
+We need to now place the built library to the location where our compiler can find it
+
+```
+cd ..
+sudo mkdir /usr/include/benchmark
+sudo cp include/benchmark/benchmark.h /usr/include/benchmark/
+sudo cp build/src/libbenchmark.a /usr/local/lib/
+```
```
Comment (dominichamon) [inline README.md]: replace all this with

`make install`

File: README.md [hunk 4]
```diff
@@ -13,6 +13,78 @@ IRC channel: https://freenode.net #googlebenchmark
...
+
+Let's test it now. We create a directory anywhere in our system
```
Comment (dominichamon) [inline README.md]: i think you can remove the test instructions as that's covered by the rest of the docs already.

Comment (Kumar-Kishan) [reply README.md]: It's 3 in the morning here. Will recommit the fix tomorrow

Comment (Kumar-Kishan) [reply README.md]: Added the changes, as per told

**Pull request merged**